### ISSUE TITLE
movementDuration

### ISSUE THREAD

**Issue by Tontonpat** [dubocr/homebridge-tahoma#257]
Bonjour Romain,
Je viens de tester la fonction movementDuration (qui n’apparaît pas dans les réglages du plugin) en indiquant le temps de fermeture de mon volet (25s) dans la config.
Je pensais que tu pouvais régler la position du volet RTS comme un IO, mais en mettant 30% ou 80% ça règle le volet en position MY.
Est ce moi qui ai mal compris cette fonction?

**Comment by Tontonpat** [comment]
Ta dernière mise à jour 2.1.5 semble régler ce problème.
Comme d’habitude, t’es au top!

**Comment by Tontonpat** [comment]
Du coup, ceux qui veulent leur position MY ne peuvent plus s’en servir (en même temps ça n’est plus d’une grande utilité)

**Comment by Tontonpat** [comment]
Je voulais aussi te féliciter et te remercier pour ton travail, de la part d’un ami qui n’a pas de compte ici.

**Comment by dubocr** [comment]
> Du coup, ceux qui veulent leur position MY ne peuvent plus s’en servir (en même temps ça n’est plus d’une grande utilité)

Effectivement, le « my » ne sera plus utilisable si on définit movementDuration. Il reste néanmoins possible sans le paramètre. Il faudra choisir mais avec movementDuration on peut faire l’équivalent du my de façon un peu moins précise c’est vrai.

merci pour ces retours et encouragements 🙏

**Comment by Tontonpat** [comment]
Une question: si le temps est supérieur à 60 secondes, est ce que çà peut poser un problème?
Il n'y a pas un délai maximum avant de pouvoir faire un "CMDCANCELLED"?
Je suis en train de tester sur mon store qui met 67 secondes.
Je te dirai

**Comment by Tontonpat** [comment]
@dubocr 
j’ai une automatisation de fermeture avec 3 volets, dont 1 que je laisse un entre ouvert (10%).
Du coup le cmdcancelled arrête également les 2 volets que je voulais fermer complètement:
![EBB49321-40C1-4151-98E3-4A5499D3946E](https://user-images.githubusercontent.com/49283285/112599387-2b2e2f00-8e10-11eb-98af-024502541f85.jpeg)

**Comment by dubocr** [comment]
Haha oui effectivement, je suis obligé de regrouper les commandes pour ne pas dépasser la limite de 10 simultanées lors des scenes mais dans ce cas il faudra que j'y déroge. Je vais regarder ça.

Pourras-tu fermer partiellement un volet sur TaHoma puis te connecter à ma page pour que je vois comment ils le gèrent.

On est daccord sur TaHoma tu ne peux pas programmer une ouverture/fermeture partielle dans un scénario ?

Pour l'histoire des 60 sec, je ne sais pas te dire je ne sais pas ce qui provoque la fin de l'action côté TaHoma.

**Comment by Tontonpat** [comment]
C’est fait

**Comment by dubocr** [comment]
Merci, peux-tu me donner les derniers chiffres de ton bridge je ne retrouve pas dans nos autres échanges.
Oui pour le MY, pas fermeture partielle je pensais sauf le MY.

**Comment by Tontonpat** [comment]
9950

**Comment by dubocr** [comment]
Merci, j'ai fait une mise à jour qui devrais améliorer tout ça.
J'ai du executer de façon indépendante les commandes pour pouvoir les annuler.
J'ai activer la fonctionnalité par défaut car la fonctionnalité "my" est désormais accessible en direct sur un switch (qui s'appelle "Alimentation" pour l'instant) via des applis comme Eve.
Tiens moi au courant.

**Comment by Tontonpat** [comment]
Merci pour la mise à jour, je test et je te dis.
Par contre au redémarrage j’ai eu le droit à ça:
![E82603FD-2C88-4314-8CE4-3324B45B74BC](https://user-images.githubusercontent.com/49283285/112761450-b881a700-8ffb-11eb-8906-4da90894b7d9.jpeg)

**Comment by dubocr** [comment]
Oui c'est la fameuse caractéristique pour my. Le message n'apparaitra plus.

**Comment by Tontonpat** [comment]
Ça a l’air d’être bon, les 2 volets qui se ferment complètement ont bien été séparés de celui qui se ferme partiellement lors de l’envoi des commandes. Je verrais demain soir. Encore merci pour tout le travail fait 👍
